docs/help/_posts/2010-02-03-return-for-any-args.markdown

```diff
@@ -17,7 +17,7 @@ ICalculator calculator;
 A call can be configured to return a value regardless of the arguments passed using the `ReturnsForAnyArgs()` extension method.
 
 ```csharp
-calculator.Add(1, 2).ReturnsForAnyArgs(100); 
```

Comment:
I wonder if we should keep this one instance of non-`default`. My reasoning is that it is the only place that makes it clear that `ReturnsForAnyArgs` will overwrite whatever arguments are specified in the call. 🤔

Reply:
This is exactly the place we introduce these methods, so it might be only doc people read about this extension 😅 

I modified sample to keep the original example, but added an explicit tip about `default`. Let me know if you like it.

Reply:
👍